content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -184,21 +184,21 @@ $ helm template install/kubernetes/helm/istio <the flags you used to install Ist
 Note that you should use the same Helm command you used [to install Istio](/docs/setup/kubernetes/helm-install),
 in particular, the same value of the `--namespace` flag. In addition to the flags you used to install Istio, add `--set global.proxy.includeIPRanges="10.0.0.1/24" -x templates/sidecar-injector-configmap.yaml`.
 
-Redeploy the _sleep_ application as described in the [Before you begin](/docs/tasks/traffic-management/egress/#before-you-begin) section.
+Redeploy the `sleep` application as described in the [Before you begin](/docs/tasks/traffic-management/egress/#before-you-begin) section.
 
-### Determine the value of `global.proxy.includeIPRanges`
+### Set the value of global.proxy.includeIPRanges
```

Comment:
Need to have the backticks around global.proxy.includeIPRange, which will solve the spelling error reported by the build process.